README.md

```diff
@@ -1,7 +1,9 @@
 # PHP mysqlnd redirection extension mysqlnd_azure
 The source code here is a PHP extension implemented using mysqlnd plugin API (https://www.php.net/manual/en/mysqlnd.plugin.php), which provides redirection feature support.  The extension is also available on PECL website at  https://pecl.php.net/package/mysqlnd_azure.
 
-**Important notice: There is a limitation that for Azure MySQL, redirection is only possible when the connection is configured with SSL, and it will only support TLS 1.2 with FIPS approved cipher for redirection.**
+**Important notice:**
```

Comment:
I chose to keep the position, but change the title to **Important notice for prerequisites to use redirection:**. it appears at the beginning of the doc, and is with bold format, so I think it may better to highlight it

Reply:
I've changed the level to same as Setup